app/controllers/concerns/api/v3/sync_to_user.rb

```diff
@@ -2,33 +2,31 @@ module Api::V3::SyncToUser
   extend ActiveSupport::Concern
 
   included do
-    def region_records
-      model = controller_name.classify.constantize
-      model.syncable_to_region(current_sync_region)
-    end
-
     def current_facility_records
-      region_records
-        .where(patient: prioritized_patients)
+      model
+        .where(patient: current_facility.syncable_patients.pluck(:id))
         .updated_on_server_since(current_facility_processed_since, limit)
     end
 
     def other_facility_records
-      other_facilities_limit = limit - current_facility_records.count
+      other_facilities_limit = limit - current_facility_records.size
+      other_patient_records =
+        current_sync_region.syncable_patients.pluck(:id) - current_facility.syncable_patients.pluck(:id)
 
-      region_records
-        .where.not(patient: prioritized_patients)
+      return model.none if other_facilities_limit.zero?
+      model
+        .where(patient: other_patient_records)
         .updated_on_server_since(other_facilities_processed_since, other_facilities_limit)
     end
 
     private
 
-    def records_to_sync
-      current_facility_records + other_facility_records
+    def model
+      controller_name.classify.constantize.for_sync
```

Comment:
I think I understand why you are returning a AR scope here instead of the actual model, but its a confusing mismatch ....maybe rename the method to something like `scope` to better represent that its the base scope for other methods to build off of?  Or maybe `region_scope`, `sync_scope`, `default_sync_scope` ?

Reply:
Yeah, I wasn't happy with continuing to calling it `model`. We'd removed `model` earlier because I disliked it's hanging presence with invisible meanings. But I think `sync_scope` makes sense 👍🏼